feat(changelog): postgresql-and-mysql-added-encryption-at-rest-general-a 2025-01-16 (#4234)
scaleway/docs-content: changelog/january2025/2025-01-16-postgresql-and-mysql-added-encryption-at-rest-general-a.mdx

---

> title: Encryption-at-Rest General Availability

Reviewer: ```suggestion
title: Encryption at rest  is now in General Availability
```

> title: Encryption at rest  is now in General Availability

Reviewer 2: ```suggestion
title: Encryption at rest is now in General Availability
```

status: added
author:
    fullname: 'Join the #database channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2025-01-16
category: databases
product: postgresql-and-mysql
---

> Encryption-at-rest is now GA for Managed PostgreSQL & MySQL. Refer to this [documentation](https://www.scaleway.com/en/docs/managed-databases/postgresql-and-mysql/api-cli/setting-up-encryption-at-rest/) to activate it, and analyse these [benchmarks](https://www.scaleway.com/en/docs/managed-databases/postgresql-and-mysql/reference-content/encryption-at-rest-performance-benchmark/)  to understand potential impacts on performances.

Reviewer: ```suggestion
Encryption at rest is now GA for Managed PostgreSQL & MySQL. Refer to the [Setting up encryption at rest documentation](https://www.scaleway.com/en/docs/managed-databases/postgresql-and-mysql/api-cli/setting-up-encryption-at-rest/) to learn how to activate it. To understand potential impacts on performance, refer to the [Encryption at rest performance benchmark](https://www.scaleway.com/en/docs/managed-databases/postgresql-and-mysql/reference-content/encryption-at-rest-performance-benchmark/) performed by our Managed Databases team. 
```
